api/src/main/java/org/pmiops/workbench/api/DataSetController.java

```diff
@@ -132,23 +132,17 @@
     } else if ((dataSetRequest.getCohortIds() == null || dataSetRequest.getCohortIds().size() == 0)
         && !includesAllParticipants) {
       throw new BadRequestException("Missing cohort ids");
-    } else if (dataSetRequest.getValues() == null || dataSetRequest.getValues().size() == 0) {
+    } else if (dataSetRequest.getDomainValuePairs() == null
+        || dataSetRequest.getDomainValuePairs().isEmpty()) {
       throw new BadRequestException("Missing values");
     }
     Timestamp now = new Timestamp(clock.instant().toEpochMilli());
     workspaceService.getWorkspaceEnforceAccessLevelAndSetCdrVersion(
         workspaceNamespace, workspaceId, WorkspaceAccessLevel.WRITER);
     long wId = workspaceService.get(workspaceNamespace, workspaceId).getWorkspaceId();
     List<DataSetValues> dataSetValuesList =
```

Comment:
NOT updating `DataSetValues`.  That is a whole other can of worms.  Aiming for minimal in this PR.